src/Sylius/Bundle/CoreBundle/Form/Extension/CountryTypeExtension.php

```diff
@@ -80,9 +80,6 @@ private function getCountryName(string $code): string
         return Countries::getName($code);
     }
 
-    /**
-     * @return array|CountryInterface[]
-     */
```

Comment:
Why did you remove `CountryInterface[]` type-hint?

Reply:
It was not right, the method returns a string[]. It could be automatically recognized by PHPStan but I anyway added it to the doc.

Reply:
Great, it is used by IDE too, so when you do foreach on an array you get type-hint :D.